### PR TITLE
[EISW-81712] Rename `VPUX` to `VPU` in openvino codebase

### PR DESCRIPTION
**Pull request by MirceaDan99**
### Details:
 - Replaced all the `VPUX` occurrences from scripts and tools of OV

### Tickets:
 - *[EISW-81712](https://jira.devtools.intel.com/browse/EISW-81712)*

### Related PRs
- [VPUX-PLUGIN-PR-4836](https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/4836)

### CODE REVIEW

File: src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp [hunk 1]
```diff
@@ -17,7 +17,7 @@ inline const std::string get_plugin_lib_name_by_device(const std::string& device
             { "HETERO", "openvino_hetero_plugin" },
             { "BATCH", "openvino_auto_batch_plugin" },
             { "MULTI", "openvino_auto_plugin" },
-            { "VPUX", "openvino_intel_vpux_plugin" },
+            { "VPU", "openvino_intel_vpux_plugin" },
```
Comment (ilya-lavrenov) [inline src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp:20]: what is the proper plugin name?

**Review by ilya-lavrenov — COMMENTED** [review]
I suppose we should also remove this folder https://github.com/openvinotoolkit/openvino/tree/master/src/inference/include/ie/vpu to avoid confusions between _new_ and _old_ VPUs :)

File: src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp (hunk 1 above)
Comment (MirceaDan99) [reply src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp:20]: We intend to use only `VPU` in the future.

Comment (MirceaDan99) [reply src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/include/api_conformance_helpers.hpp:20]: At the moment we use a hybrid scenario where both `VPU` and `VPUX` devices are recognized, but `VPU` is a [pseudo plugin](https://github.com/intel-innersource/applications.ai.vpu-accelerators.vpux-plugin/pull/4549) for `VPUX` using the same `openvino_intel_vpux_plugin` library. However, in the future we plan to accept only `VPU` device, so library name should also be changed.

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**